### ISSUE TITLE
Packing Multi-RID Multi-TFM project overwrites PDBs with PDB for wrong TFM.

### ISSUE THREAD

**Issue by jkoritzinsky** [onovotny/MSBuildSdkExtras#171]
While working on https://github.com/SharpGenTools/SharpGenTools, I've noticed that while using the multi-RID, multi-TFM support and the ref assembly support in this SDK, that packing the nuget package using `dotnet pack` will result in PDBs for one TFM overwriting the PDBs for another TFM.

Pack the SharpGen.Runtime project from https://github.com/SharpGenTools/SharpGenTools/tree/4edbb17a75ba25cb7b1792a3178f051f9c0618cd and you'll see that the PDBs are not copied correctly into the package, breaking symbol resolution (and code-coverage, which is how I found this).

**Comment by jkoritzinsky** [comment]
Adding the following target to clean up the bad entries before generating the package fixes the issue:
```
  <Target Name="RemoveInvalidDuplicateSymbols" BeforeTargets="GenerateNuspec" AfterTargets="_GetPackageFiles">
    <ItemGroup>
      <_PackageFiles Remove="@(_PackageFiles)" Condition="'%(_PackageFiles.PackagePath)' != 'runtimes/%(_PackageFiles.Rid)/lib/%(_PackageFiles.TargetFramework)' and '%(_PackageFiles.TargetPath)' == '$(AssemblyName).pdb'" />
    </ItemGroup>
  </Target>
```

**Comment by onovotny** [comment]
Any chance you can submit a PR with this?

**Comment by jkoritzinsky** [comment]
I'll try to get a PR out. I want to see if I can get a cleaner implementation though than just cleaning up the bad entries after the back.

**Comment by jkoritzinsky** [comment]
Fix is in #172. Much simpler than I thought it was going to be.

**Comment by onovotny** [comment]
2.0.31 is rolling out to NuGet with this merged in, should be visible there in the next 10-15 min. Thanks for the fix!
